appcenter-push/Push.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface PushNotification {
+    message?: string | null;
+    title?: string | null;
+    customProperties?: { [name: string]: stirng };
+}
+
+export interface ListenerMap {
```

Comment:
Please use our Android SDK name convention.

Reply:
Just done the typescript definitions, not the backing js (`setListener(listenerMap)`)